Source/WTF/wtf/PlatformHave.h

```diff
@@ -1018,6 +1018,14 @@
 #define HAVE_PER_APP_ACCESSIBILITY_PREFERENCES 1
 #endif
 
+#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000) \
```

Comment:
We only need the macOS check here, nothing else (including macCatalyst on old macOS) is supported.